app/views/registration_wizard/work_setting.html.erb

```diff
@@ -1,34 +1,7 @@
-<% content_for :title do %>
-  <%= @form.errors.present? ? "Error: " : nil %><%= t("helpers.legend.registration_wizard.work_setting") %>
-<% end %>
-
-<% content_for :before_content do %>
-  <%= render GovukComponent::BackLinkComponent.new(
-    text: "Back",
-    href: registration_wizard_show_path(@wizard.previous_step_path)
-  ) %>
-<% end %>
-
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <%= form_with model: @form, url: registration_wizard_form_url(@form), scope: 'registration_wizard', method: :patch do |f| %>
-      <%= f.govuk_error_summary %>
-
-      <%= f.govuk_radio_buttons_fieldset(:work_setting,
-                                         legend: {
-                                           tag: "h1",
-                                           size: "xl"
-                                         }
-                                        ) do %>
-        <%= f.govuk_radio_button(:work_setting, "early_years_or_childcare", link_errors: true) %>
-        <%= f.govuk_radio_button(:work_setting, "a_school") %>
-        <%= f.govuk_radio_button(:work_setting, "an_academy_trust") %>
-        <%= f.govuk_radio_button(:work_setting, "a_16_to_19_educational_setting") %>
-        <%= f.govuk_radio_divider %>
-        <%= f.govuk_radio_button(:work_setting, "other") %>
-      <% end %>
-
-      <%= f.govuk_submit %>
-    <% end %>
-  </div>
-</div>
+<%=
+  render(
+    'registration_wizard/shared/question_page',
+    form: @form,
+    wizard: @wizard,
+  )
+%>
```

Comment:
Love it, this is so nice to see spreading around the other questions. 😁

Reply:
Yeah will try to do as many as possible although still few bits that im confused with how the magic sets up everything.